test/unit/translations/maps/linkedPhysicalIndexToValueMap.spec.js

```diff
@@ -0,0 +1,217 @@
+import { LinkedPhysicalIndexToValueMap as IndexToValueMap } from 'handsontable/translations';
```

Comment:
I think that the unit test for that kind of IndexMap could be improved. I've found that some methods aren't tested very well or not tested at all. For example, I removed `getLength`, `insert` and `remove` methods from the LinkedMap, and run test specs defined by that particular file - all test passes (the error was caught within IndexMapper specs). When I removed only the `getLength` method all unit test passes.

Generally speaking, I'd add a unit test for each method separately by checking the internal state after execution. This collection is especially important as the new internal state `orderOfIndexes` is changing the Map abstract behavior.

Reply:
Thanks for the critical remarks 👍 The generic `getLength` will work, but it's slower. That's the reason why it's overridden. Thus, removing it won't give effects in unit tests. However, I've added an extra check for it.